sql/analyzer/validation_rules.go

```diff
@@ -464,6 +464,9 @@ func validateOperands(ctx *sql.Context, a *Analyzer, n sql.Node, scope *Scope, s
 
 	// We do not use plan.InspectExpressions here because we're treating
 	// top-level expressions of sql.Node differently from subexpressions.
+	if _, ok := n.(*plan.CreateTrigger); ok {
```

Comment:
(minor) It would be good to raise this up a couple of lines in the file and give it a quick comment about why we're skipping operand validation for create trigger statements. That comment would help future code readers understand the exception, and it would also put the comments on line 465 and 466 closer to the `transform.Inspect` line that they are referring to.